eng/pipelines/common/push-nightly.yml

```diff
@@ -55,7 +55,7 @@ stages:
           - task: DownloadPipelineArtifact@2
             inputs:
               artifactName: vs-msi-nugets
```

Comment:
This artifact name will need to be changed, as it is the one that has been removed. It is also probably worth removing the nuget-signed artifact download above, or adding a filter to this to reduce duplicates:
```yaml
  patterns: |
    **/*msi*.nupkg
```

Or a third option would be to download from the drop services:
```yaml
- task: ms-vscs-artifact.build-tasks.artifactDropDownloadTask-1.artifactDropDownloadTask@1
  displayName: Download path/to/drop/nugets
  inputs:
    dropServiceURI: https://devdiv.artifacts.visualstudio.com/DefaultCollection
    buildNumber: path/to/drop/nugets
    destinationPath: $(Build.StagingDirectory)\nuget-signed
```

Reply:
Should we go with this ?